### DISCUSSION TITLE
蘋果AirPods Pro 2將開賣

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#743] [General]
AirPods Pro 2除了提升續航力之外
調整音訊的演算法也升級了
價格同時也比一代低
果粉們可以考慮一下了！
https://news.tvbs.com.tw/tech/1924574

**Comment by AnAnistired** [comment]
先說我用蘋果手機，但airpods 除了可以當追蹤器外，其他真的都比不上他牌耳機，又賣超貴，垃圾就是垃圾

**Comment by williamyen0623** [comment]
埃爾帕茲破真的好用

**Comment by Pietra1226** [comment]
希望AirPods pro1可以降價，再買一個來用

**Comment by danwan1027** [comment]
希望他的價格可以再便宜一點

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
以同價位來說 蘋果的兒機真的不算是cp值高的 降個價的話會更香

**Comment by CPCpyt** [comment]
心動了，立刻手刀購買！
不過弄丟真的新好痛QQ
